src/avalanchemq/http/controller/prometheus.cr

```diff
@@ -172,7 +172,7 @@ module AvalancheMQ
           unacked += d[:messages_unacknowledged]
           connections += vhost.connections.size
           vhost.connections.each do |conn|
-            channels += conn.channels.size
+            channels += conn.channel_count
           end
           consumers += vhost.consumers.size
```

Comment:
Hidden slow code here as well: consumers is a method and does this: 
```
def consumers
  @connections.flat_map { |conn| conn.channels.each_value.flat_map &.consumers }
end
```

I guess it was this that ate all the RAM

Reply:
ok! yes!